compiled_starters/typescript/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "@codecrafters/build-your-own-grep",
+    "description": "Build your own Grep challenge, from CodeCrafters",
+    "type": "module",
+    "scripts": {
+        "dev": "bun run app/main.ts"
+    },
+    "dependencies": {
+        "fs-extra": "^11.2.0"
```

Comment:
`fs-extra` doesn't seem like it's be useful for grep, but yeah can't think of anything else at the moment